packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -208,6 +220,59 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
     [adapter]
   );
 
+  /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+  const onInlineImageClicked = useCallback(
+    async (attachment: FileMetadata, imageName?: string, senderId?: string): Promise<void> => {
```

Comment:
I don't think we need to pass the `senderId` into this method. We only use it internally for the `AvatarPersona` is it possible to just reuse the `OnRenderAvatarCallback`?

Reply:
> I don't think we need to pass the `senderId` into this method. We only use it internally for the `AvatarPersona` is it possible to just reuse the `OnRenderAvatarCallback`?

The OnRenderAvatarCallback also takes a `userId` prop which is the sender's id. We are at the Chat composite level and we don't know anything about the message that's been clicked (including the sender) unless it's passed in with the onInlineImageClicked callback